packages/govern-server/package.json

```diff
@@ -17,7 +17,8 @@
     "ts-node": "^9.0.0"
   },
   "devDependencies": {
-    "typescript": "^3.8.3",
-    "@types/express": "^4.17.8"
+    "@types/express": "^4.17.8",
+    "ts-node-dev": "^1.0.0",
+    "typescript": "^4.0.3"
```

Comment:
We shouldn't forget to keep the TS versions anywhere in-sync. Look for example at the root ``package.json``.

Reply:
Wonder if there's a "nicer" way to keep this all in sync?

Reply:
`yarn upgrade-interactive --latest` should be enough I think? It groups all the workspaces dependencies, so we can select all the `typescript` for example, and they will upgrade at once.